dsa-solutions/lc-solutions/0000-0099/0004-Median-of-two-Sorted-Array.md

```diff
@@ -2,19 +2,19 @@
 id: median-of-two-sorted-arrays
 title: Median of Two Sorted Arrays(Leetcode)
 sidebar_label: 0004 - Median of Two Sorted Arrays
-tags: 
-    - Array
-    - Binary Search
-    - Divide and Conquer0
-    - LeetCode
+tags:
+  - Array
+  - Binary Search
+  - Divide and Conquer0
+  - LeetCode
 description: Given two sorted arrays nums1 and nums2 of size m and n respectively, return the median of the two sorted arrays. The overall run time complexity should be $O(log (m+n))$.
```

Comment:
remove `The overall run time complexity should be $O(log (m+n))$.` because this is not a good practice